### DISCUSSION TITLE
Qodana action not adding a link to the full report

### DISCUSSION THREAD

**Discussion by dennisdoomen** [JetBrains/Qodana#200] [Q&A] [answered]
We've been piloting Qodana for a while now, but recently something changed. Qodana still reports the problems it finds in the PR, but it doesn't offer a direct link to the analysis on Qodana Cloud. It used to do that. Did we misconfigure something? If I go to Qodana Cloud, I _can_ manually find the corresponding branch, so the report _is_ uploaded.

See https://github.com/fluentassertions/fluentassertions/pull/2359#issuecomment-1745572995

**Comment by brichbash** [comment, marked as answer]
Hello,

It's a known problem connected to the native mode in Qodana. Thanks for reporting. We're working on it.
